jgrapht-core/src/main/java/org/jgrapht/graph/Subgraph.java

```diff
@@ -207,7 +210,6 @@ public E addEdge(V sourceVertex, V targetVertex)
         for (E e : edges) {
```

Comment:
I can't seem to put this comment in the right place... you can only comment on modified code. Sorry about that.